.github/workflows/ci-update-workflow.yml

```diff
@@ -0,0 +1,62 @@
+name: ci-update-workflow
+
+on:
+  schedule:
+  # Run every hour.
+  - cron:  '0 * * * *'
+
```

Comment:
Lets define the default token permission for the jobs in this action 
```
# Declare default permissions as read only.
permissions: read-all
```
like in https://github.com/GoogleCloudPlatform/functions-framework-go/blob/master/.github/workflows/scorecard.yml